docs/application/dotnet/guides/nui/view.md

```diff
@@ -1,75 +1,133 @@
 # View
 
-The `View` class is the base class for all views. With this class you can manage the background color and images for UI components.
+[View](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html) is the fundamental concept for the all components such as `Button`, `Image`, `Text`, and so on.
+`View` provides [Properties](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#properties), [Methods](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#methods), and [Events](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#events) that are commonly used on every components. To render `View`, you must add it on [Window](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.Window.html) as a child.
 
-In this tutorial, the following subjects are covered:
-
-[View events](#1)<br>
-[Setting the Background Color](#2)<br>
-[Setting the Background Image](#3)<br>
 
 <a name="1"></a>
-## View events
+## Transforms
 
-The following table lists the basic event provided by the `View` class:
+`Position`, `Orientation`, and `Scale` of `View` are known as transforms.
+`View` supports hierarchical structure. Therefore, `Position`, `Orientation`, and `Scale` of every `View` are affected by their parents `Views`.
```

Comment:
Can we change "their parents `Views`" to "their parent's `View`"?

Reply:
I think `Views` is not necessary. Could I remove it? just 

Therefore, `Position`, `Orientation`, and `Scale` of every `View` are affected by their parents.